packages/asset-service/src/service/descriptions/es.json

```diff
@@ -1,7 +1,8 @@
 {
+  "eip155:1/slip44:60": "Ethereum es una [plataforma de contratos inteligentes](https://www.coingecko.com/en/categories/smart-contract-platform) que permite a los desarrolladores crear tokens y aplicaciones descentralizadas (dapps). ETH es la moneda nativa de la plataforma Ethereum y también funciona como tarifa de transacción para los mineros en la red Ethereum.<br/><br/>Ethereum es el pionero en contratos inteligentes basados en blockchain. El contrato inteligente es esencialmente un código de computadora que se ejecuta exactamente según lo programado sin posibilidad de tiempo de inactividad, censura, fraude o interferencia de terceros. Puede facilitar el intercambio de dinero, contenido, propiedades, acciones o cualquier cosa de valor. Cuando se ejecuta en la cadena de bloques, un contrato inteligente se convierte en un programa informático autónomo que se ejecuta automáticamente cuando se cumplen condiciones específicas.<br/><br/>Ethereum permite a los programadores ejecutar contratos inteligentes completos que pueden personalizarse. En lugar de brindar un conjunto de operaciones limitadas, Ethereum les permite a los desarrolladores tener un control completo sobre la personalización de su contrato inteligente, lo que les brinda el poder de crear aplicaciones únicas e innovadoras.<br/><br/>Ethereum es el primer [smart plataforma de contrato] (https://www.coingecko.com/en/categories/smart-contract-platform), han ganado mucha popularidad, lo que ha dado lugar a que nuevos competidores luchen por la cuota de mercado. Los competidores incluyen: [Ethereum Classic](https://www.coingecko.com/en/coins/ethereum-classic) que es la antigua cadena de Ethereum, [Qtum](https://www.coingecko.com/en/monedas/qtum), [EOS](https://www.coingecko.com/en/coins/eos), [Neo](https://www.coingecko.com/en/coins/neo), [ICON](https://www.coingecko.com/en/coins/icon), [Tron](https://www.coingecko.com/en/coins/tron) y [Cardano](https://www.coingecko.com/en/coins/cardano).<br/><br/>Las billeteras Ethereum son bastante simples de configurar con múltiples opciones populares como [KeepKey](https://www.coingecko.com/learn/keepkey-bitcoin-hardware-wallet-review) y [MetaMask](https://www.coingecko.com/buzz/complete-beginners-guide-to-metamask). Lea aquí para obtener más guías sobre el uso de la billetera Ethereum: [Cómo usar una billetera Ethereum](https://www.coingecko.com/buzz/how-to-use-an-ethereum-wallet)",
```

Comment:
Whitespace still present in `[smart plataforma de contrato] (https://www.coingecko.com/en/categories/smart-contract-platform)`

```suggestion
  "eip155:1/slip44:60": "Ethereum es una [plataforma de contratos inteligentes](https://www.coingecko.com/en/categories/smart-contract-platform) que permite a los desarrolladores crear tokens y aplicaciones descentralizadas (dapps). ETH es la moneda nativa de la plataforma Ethereum y también funciona como tarifa de transacción para los mineros en la red Ethereum.<br/><br/>Ethereum es el pionero en contratos inteligentes basados en blockchain. El contrato inteligente es esencialmente un código de computadora que se ejecuta exactamente según lo programado sin posibilidad de tiempo de inactividad, censura, fraude o interferencia de terceros. Puede facilitar el intercambio de dinero, contenido, propiedades, acciones o cualquier cosa de valor. Cuando se ejecuta en la cadena de bloques, un contrato inteligente se convierte en un programa informático autónomo que se ejecuta automáticamente cuando se cumplen condiciones específicas.<br/><br/>Ethereum permite a los programadores ejecutar contratos inteligentes completos que pueden personalizarse. En lugar de brindar un conjunto de operaciones limitadas, Ethereum les permite a los desarrolladores tener un control completo sobre la personalización de su contrato inteligente, lo que les brinda el poder de crear aplicaciones únicas e innovadoras.<br/><br/>Ethereum es el primer [smart plataforma de contrato](https://www.coingecko.com/en/categories/smart-contract-platform), han ganado mucha popularidad, lo que ha dado lugar a que nuevos competidores luchen por la cuota de mercado. Los competidores incluyen: [Ethereum Classic](https://www.coingecko.com/en/coins/ethereum-classic) que es la antigua cadena de Ethereum, [Qtum](https://www.coingecko.com/en/monedas/qtum), [EOS](https://www.coingecko.com/en/coins/eos), [Neo](https://www.coingecko.com/en/coins/neo), [ICON](https://www.coingecko.com/en/coins/icon), [Tron](https://www.coingecko.com/en/coins/tron) y [Cardano](https://www.coingecko.com/en/coins/cardano).<br/><br/>Las billeteras Ethereum son bastante simples de configurar con múltiples opciones populares como [KeepKey](https://www.coingecko.com/learn/keepkey-bitcoin-hardware-wallet-review) y [MetaMask](https://www.coingecko.com/buzz/complete-beginners-guide-to-metamask). Lea aquí para obtener más guías sobre el uso de la billetera Ethereum: [Cómo usar una billetera Ethereum](https://www.coingecko.com/buzz/how-to-use-an-ethereum-wallet)",
```